ngraph/test/models/onnx/softmax_axis_1.prototxt

```diff
@@ -52,5 +52,5 @@ graph {
   }
 }
 opset_import {
-  version: 7
+  version: 6
```

Comment:
Do we still have tests for old softmax?

Reply:
Yes, that is why I changed opset version from 7 to 6.